src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -68,13 +75,24 @@ static String stringOrNull(RequestParameter requestParameter) {
     return requestParameter == null ? null : requestParameter.getString();
   }
 
+  private static JsonObject jsonObjectRemoveNull(JsonObject obj) {
```

Comment:
The method name indicates that the input JsonObject is altered. This is fine for the usage in this class.

However, the implementation creates a copy in a completely new JsonObject. If this really is intended the method name should be changed, for example `copyWithoutNulls`.

Reply:
If it doesn't create a new JsonObject that method might use `void` as return type.

Reply:
making a copy